tests/Controller/CRUDControllerTest.php

```diff
@@ -3219,36 +3202,6 @@ public function testBatchActionActionInvalidCsrfToken(): void
         }
     }
 
-    /**
-     * NEXT_MAJOR: Remove this legacy group.
```

Comment:
Here it says to just remove the `@group legacy` part, not sure if the test should be removed.

Reply:
Restored